### PR TITLE
implementing episode selected to homepage

### PR DESCRIPTION
**Pull request by garySiladi**
This PR implements the episode selected component

### CODE REVIEW

File: src/components/episode-selected/episode-selected.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // @flow
 import React from 'react';
-import { Link } from 'react-router';
+import classnames from 'classnames';
+import { browserHistory, Link } from 'react-router';
```
Comment (varholak-peter) [inline src/components/episode-selected/episode-selected.js]: We want to keep react import together so move this one by one line

File: src/components/episode-selected/episode-selected.js [hunk 2]
```diff
@@ -9,27 +10,114 @@ type EpisodeSelectedType = {
...
+  state: {
+    selectedItem: number,
+  }
+  componentDidMount() {
+    document.addEventListener('keydown', this.handleKeyPress);
+  }
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleKeyPress);
+  }
+  handleKeyPress(event: Object) {
```
Comment (varholak-peter) [inline src/components/episode-selected/episode-selected.js]: `event: Event`

File: src/components/episode-selected/episode-selected.js [hunk 3]
```diff
@@ -9,27 +10,114 @@ type EpisodeSelectedType = {
...
+      url,
+      subtitle,
+      title,
+      description,
+    } = this.props;
+    const {
+      selectedItem,
+    } = this.state;
+    const watchString : string = `/watch?id=${id}`;
+    const learnString : string = `/learn?id=${id}`;
```
Comment (varholak-peter) [inline src/components/episode-selected/episode-selected.js]: are these two types needed? I think flow should be able to assume them properly.

Comment (garySiladi) [inline src/components/episode-selected/episode-selected.js]: it was the initial implementation of the previous dev working on this 😄 
I will check

**Pull request merged**